### DISCUSSION TITLE
Italic Text not working in node

### DISCUSSION THREAD

**Discussion by RathodAnnu13** [fabricjs/fabric.js#8964] [Q&A]
Hi, I'm new to fabric.js. Currently facing some problem as showing in following.

Node version: 16.13.0
FabricJS version: 5.3.0 
JSON: {"canvasJSON": {
                    "objects": [
                        {
                            "angle": 0,
                            "backgroundColor": "",
                            "charSpacing": 0,
                            "direction": "ltr",
                            "fill": "#e72222",
                            "fillRule": "nonzero",
                            "flipX": false,
                            "flipY": false,
                            "fontFamily": "Roboto-Regular",
                            "fontSize": 51,
                            "fontStyle": "italic",
                            "fontWeight": "bold",
                            "globalCompositeOperation": "source-over",
                            "height": 57.63,
                            "left": 11,
                            "lineHeight": 1.16,
                            "linethrough": false,
                            "opacity": 1,
                            "originX": "left",
                            "originY": "top",
                            "overline": false,
                            "paintFirst": "fill",
                            "path": null,
                            "pathAlign": "baseline",
                            "pathSide": "left",
                            "pathStartOffset": 0,
                            "scaleX": 1,
                            "scaleY": 1,
                            "shadow": {
                                "affectStroke": false,
                                "blur": 0,
                                "color": "#000000",
                                "nonScaling": false,
                                "offsetX": 0,
                                "offsetY": 0
                            },
                            "skewX": 0,
                            "skewY": 0,
                            "stroke": "#ffffff",
                            "strokeDashArray": null,
                            "strokeDashOffset": 0,
                            "strokeLineCap": "butt",
                            "strokeLineJoin": "miter",
                            "strokeMiterLimit": 10,
                            "strokeUniform": false,
                            "strokeWidth": 0,
                            "styles": [],
                            "text": "ADD HEADING TEXT ITALIC",
                            "textAlign": "left",
                            "textBackgroundColor": "",
                            "top": 137,
                            "type": "i-text",
                            "underline": false,
                            "version": "5.3.0",
                            "visible": true,
                            "width": 681.53
                        }
                    ],
                    "background": "#d9d9d9"
                }}
 I'm trying to create canvas from this JSON.
 but italic font is not working.

thanks for answering!

**Comment by asturur** [comment]
in node you have to load the fonts manually.
Did you check: http://fabricjs.com/fabric-intro-part-4#node for the custom fonts part?

**Reply by RathodAnnu13** [reply]
Yes.. 
I am registering Fonts first.
If FontFamily is itself italic ,it is working as expected.
but when we apply style 'Italic', it isn't working.
